src/components/core/Markdown/Markdown.tsx

```diff
@@ -19,6 +19,8 @@ export default function Markdown({ text }: Props) {
             <>{children}</>
           ),
       }}
+      allowedElements={['a', 'strong', 'em', 'ol', 'ul', 'li', 'p']}
+      unwrapDisallowed
```

Comment:
this allows disallowed markdown to be rendered as text instead of ignored completely